### DISCUSSION TITLE
Trying to run Nav2 in ROS2 Galactic. LED going red (asking for a restart)

### DISCUSSION THREAD

**Discussion by AmduX64** [iRobotEducation/create3_docs#374] [Questions]
Good afternoon!!! 
Our team is currently trying to run the Navigation2 module in our Irobot Create 3. Our main issue is that we get stuck at: 
"timed out waiting for transform from base_link to odom to become available, tf error: invalid frame id "odom" passed to can transform argument target_frame - frame does not exist"

The funny thing is that sometimes it solves itself and works. But most of the times it gets stuck and the Create 3 crashes, staying red and we have to restart it manually.

We played around with the params_file and nothing has changed yet. Are we missing something?

**Comment by shamlian** [comment]
Hi there, I think we need more info about your robot's software version, networking environment, middleware configuration (both on your robot and your computer), at least, before we can help. Can you tell us more?
